hazelcast-sql/src/test/java/com/hazelcast/jet/sql_nightly/SqlSTSInnerNonEquiJoinFaultToleranceStressTest.java

```diff
@@ -16,32 +16,20 @@
 
 package com.hazelcast.jet.sql_nightly;
 
-import com.hazelcast.test.HazelcastParametrizedRunner;
-import com.hazelcast.test.HazelcastSerialParametersRunnerFactory;
 import com.hazelcast.test.annotation.NightlyTest;
-import com.hazelcast.test.annotation.ParallelJVMTest;
 import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
 
-@RunWith(HazelcastParametrizedRunner.class)
-@Parameterized.UseParametersRunnerFactory(HazelcastSerialParametersRunnerFactory.class)
-@Category({NightlyTest.class, ParallelJVMTest.class})
+@Category(NightlyTest.class)
 public class SqlSTSInnerNonEquiJoinFaultToleranceStressTest extends SqlSTSInnerEquiJoinFaultToleranceStressTest {
 
-    public SqlSTSInnerNonEquiJoinFaultToleranceStressTest() {
-        super();
-        this.sinkCount = 200;
-    }
-
     @Override
     protected String setupFetchingQuery() {
-        expectedEventsCount = eventsToProcess - 1; // we do expected fewer items for query below
-        firstItemId = 2;                              // we do expect first item to be [1, value-2]
+        expectedEventsCount = eventsToProcess - 1; // we expect fewer items for query below
+    firstItemId = 2;                               // we do expect first item to be [1, value-2]
```

Comment:
The indentation is wrong.
```suggestion
        firstItemId = 2;                           // we expect first item to be [1, value-2]
```